LayoutTests/TestExpectations

```diff
@@ -6843,6 +6843,9 @@ imported/w3c/web-platform-tests/scroll-animations/scroll-timelines/two-animation
 imported/w3c/web-platform-tests/scroll-animations/scroll-timelines/two-animations-attach-to-same-scroll-timeline.html [ Skip ]
 imported/w3c/web-platform-tests/scroll-animations/view-timelines/range-boundary.html [ Skip ]
 
+# EWS can't run this test because AsyncOverflowScrolling is disabled
+imported/w3c/web-platform-tests/pointerevents/pointerevent_hit_test_scroll_visible_descendant.html [ Skip ]
```

Comment:
That probably means no Interop points since wpt.fyi runs Ventura :(

Reply:
AsyncOverflowScrolling is not Sonoma only (UI-side compositing is, but we do both code paths in this PR so we should be ok for Interop).

Reply:
So why do we skip this test on Ventura? If this fails on Ventura, it'll fail on wpt.fyi

Reply:
We skip everywhere here. (because for this to run on EWS, we first need to activate AsyncOverflow which will be done in another PR)